pages/agent/v3/hooks.md

```diff
@@ -206,3 +206,6 @@ An example of a Windows `environment.bat` hook:
 ECHO "--- \:house_with_garden\: Setting up the environment"
 SET GITHUB_RELEASE_ACCESS_KEY='xxx'
 ```
+### Hooks on hosted agents
+
+Currently, hooks located on the agent are not available on hosted agents. Instead, we recommend using plugin or repository based hooks.
```

Comment:
Promote heading (as suggested in my comment above) and link through to relevant sections of the docs towards the top of this page (for better context).

```suggestion
## Hooks on hosted agents

Currently, hosted agents do not support [agent hooks](#hook-locations-agent-hooks). Instead, use either [repository](#hook-locations-repository-hooks)- or [plugin](#hook-locations-plugin-hooks)-based hooks.
```